toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/tools/test_xdd.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from ..conftest import XnvmeDriver, xnvme_parametrize
+
+
+def test_xdd(cijoe):
+    err, _ = cijoe.run("dd if=/dev/random of=test.0 count=2 bs=1M")
+    assert not err
+
+    commands = [
+        (
+            "xdd sync --data-input test.0 --data-output test.1 --data-nbytes 2097152",
+            "cmp test.0 test.1 && rm test.1",
+        ),
+        (
+            "xdd sync --data-input test.0 --data-output test.1 --data-nbytes 1048576 --offset 1048576",
+            "cmp test.0 test.1 --ignore-initial=1048576:0 && rm test.1",
```

Comment:
Very minor thing, but you could do `cmp test.0 test.1 --ignore-initial=1048576:0; rm test.1` if we never want to keep the file. Otherwise looks good.

Reply:
Or maybe you want the exit-code? Nevermind.